packages/examples/kitchen-sink/src/index.js

```diff
@@ -1,6 +1,7 @@
 import styles from './styles.css';
 import parcel from 'url:./parcel.webp';
 import {message} from './message';
+import warehouse from 'url:./image.jpg?height=200&as=webp';
```

Comment:
Throws `Error: Unknown pipeline webp` is this intended behaviour or a bug in named pipelines?
cc @devongovett

Reply:
caching bug?

Reply:
you're right, adding `--no-cache` fixes it